internal/pkg/image/driver/imagedriver.go

```diff
@@ -70,15 +76,19 @@ func InitImageDrivers(register bool, unprivileged bool, fileconf *apptainerconf.
 	}
 
 	var squashFeature fuseappsFeature
+	var ext3Feature fuseappsFeature
 	var overlayFeature fuseappsFeature
 	squashFeature.init("squashfuse", "mount SIF", desiredFeatures&image.ImageFeature)
+	if squashFeature.cmdPath != "" {
```

Comment:
No, because the init function initializes cmdPath so ext3Feature.cmdPath would always be empty at that point.  The reason for this is that if there's no squashfuse I don't want to even offer the imagedriver ImageFeature, since squashfuse is the most important aspect of that.  If there were separate imagedriver flags for each of the different types of image mounts I wouldn't mind.  Do you disagree with that and think it's better to offer the driver even if only ext3 image mounts are supported?  If not, I should probably explain better with a comment.

Reply:
It depends if we want to support the case to run old images that were built as a single ext3 filesystem, no strong opinion on that

Reply:
Ok that's something I hadn't thought of, changed it.